dev-support/Jenkinsfile

```diff
@@ -59,8 +59,8 @@ pipeline {
     ASF_NIGHTLIES_BASE_ORI = "${ASF_NIGHTLIES}/hbase/${JOB_NAME}/${BUILD_NUMBER}"
     ASF_NIGHTLIES_BASE = "${ASF_NIGHTLIES_BASE_ORI.replaceAll(' ', '%20')}"
     // These are dependent on the branch
-    HADOOP3_VERSIONS = "3.3.5,3.3.6,3.4.0,3.4.1"
-    HADOOP3_DEFAULT_VERSION = "3.4.1"
+    HADOOP3_VERSIONS = "3.3.5,3.3.6,3.4.0,3.4.2"
```

Comment:
I've created a separated issue for this. We can discuss whether to include more versions there.

Reply:
Reverted the Jenkinsfile change.